scripts/unused_blobs

```diff
@@ -1,26 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
Is there a reason for removing this script? This actually helped us debug a problem with an updated package name as recently as last week.

Reply:
The goal is for this script is to come back as a linter (https://github.com/cloudfoundry/wg-app-platform-runtime-ci/tree/main/diego-release/linters) instead so that it can be run on every commit instead of having it be a one-off thing